CONTRIBUTING.md

```diff
@@ -258,9 +258,8 @@ To locally lint your files using the same workflow as our CI:
 
 If you end up adding a new official Bevy crate to the `bevy` repo:
 
```

Comment:
We still need to make sure the other metadata is correct -  at `github.com/bevyengine/bevy` as the url, etc.

(For example, `benches` doesn't have that which is probably *fine*)

Reply:
Thats fair, but also a bit tangential. Feel free to create a new PR / issue :smile: